### DISCUSSION TITLE
Configuration for linking multiple OIDC's to one identity?

### DISCUSSION THREAD

**Discussion by zikes** [ory/kratos#1262] [Q&A] [answered]
I'd like to be able to allow the user to log in via one OIDC, then link 1 or more OIDCs to that identity. Currently the issue I'm running into is that in order to begin the Login flow, the redirect to `/self-service/login/browser` to acquire the `flow` token recognizes that the user is already logged in and redirects them to the default return URL without a `flow` token.

**Comment by aeneasr** [comment, marked as answer]
To link another user, point them towards the settings flow. There are options to link and unlink from SSO providers
